dotnet/src/Connectors/Connectors.Memory.AzureCognitiveSearch/AzureCognitiveSearchRecord.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Azure.Search.Documents.Indexes;
+using Azure.Search.Documents.Indexes.Models;
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.AzureCognitiveSearch;
+
+public class AzureCognitiveSearchRecord
```

Comment:
XMLDoc throughout would be lovely.

Reply:
I added to all the public methods and to the non-obvious ones. Let me know if I missed some :-)